protected/humhub/helpers/DbHelper.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace humhub\helpers;
+
+use Yii;
+use yii\db\Command;
+use yii\db\Query;
+
+/**
+ * @since 1.15
+ */
+class DbHelper
```

Comment:
I understand that it makes not so much of a difference. I personally prefer not having to write the same thing over and over again. But it's a different story if you have to maintain an API. I guess it's mainly useful in tests and migrations. Otherwise I've not been using these types of queries much, yet. Maybe that's why I find them cumbersome for the basic insert, update, and delete actions.

Now I have "reverted" the move of the methods into a separate class. That's probably not exactly what you wanted, but right now I do not have the capacity to re-write the places where it has been used. I also need to adapt my module.

"Reverted" because I've placed them now into the trait as some methods there rely on this.

Reply:
@martin-rueegg Ok, understand. As a workaround, can we mark these methods as deprecated. Then it is at least clear that we do not want to maintain them in the long term.